simulation/package.xml

```diff
@@ -14,8 +14,12 @@
   <maintainer email="astrobee-fsw@nx.arc.nasa.gov">
     Astrobee Flight Software
   </maintainer>
-  <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>roscpp</build_depend>
+  <build_depend condition="$ROS_VERSION == 1">catkin</build_depend>
```

Comment:
didn't we decide to get rid of this, and use the ros2 in the ros2 branch and ros1 in develop?

Reply:
this is actually something that could go to ros-common, unless you think it's too cluttered